app/templates/gentelella/admin/event/tickets/add_order.html

```diff
@@ -74,10 +74,10 @@
                 <strong>Payment Type: </strong>
                 <select name="payment_via" class="form-control" id="payment-type" required style="margin-left: 5px;">
                     <option value="">Select payment type</option>
-                    <option value="cheque">Cheque</option>
-                    <option value="transfer">Telegraphic Transfer (TT) / Bank Transfer</option>
-                    <option value="onsite">Onsite / Cash</option>
-                    <option value="stripe">Stripe</option>
+                    {% if event.pay_by_cheque %}<option value="cheque">Cheque</option>{% endif %}
```

Comment:
@aditya1702 this page is for the organizer. The organizer has the freedom to choose from all the options.